pallets/time-release/src/benchmarking.rs

```diff
@@ -72,8 +72,6 @@ benchmarks! {
 			schedule.start = i.into();
 			TimeReleasePallet::<T>::transfer(RawOrigin::Signed(from.clone()).into(), to_lookup.clone(), schedule.clone())?;
 		}
-
-		frame_system::Pallet::<T>::set_block_number(schedule.end().unwrap() + 1u32.into());
```

Comment:
why are you removing this? Isn't this to alter what is going to happen in the benchmark?

Reply:
It does not do anything. This pallet does not depend on the system block number.

Reply:
it definitely does, when trying to lazily clean up the storage if everything is unlocked

Reply:
This benchmark does not is what I mean to say.